style.css

```diff
@@ -558,3 +572,431 @@ ul li {
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: rgb(255, 255, 255);
+}
+
+.contact-btn:hover {
+  background-color: rgb(54, 179, 127);
+  cursor: pointer;
+}
+
+.contact-btn:focus {
+  background: rgb(13, 53, 36);
+}
+
+.contact-btn:active {
+  background: rgb(17, 209, 129);
+}
+
+.socialicon1 {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
+.socialicon li a {
+  flex: 1;
+  align-items: center;
+}
+
+.email-address {
+  width: 375px;
+  height: 189px;
+  margin-top: 17px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: rgb(66, 82, 110);
+}
+
+.last-div {
+  height: 4px;
+  width: 200px;
+  background-color: rgb(10, 11, 14);
+  border-radius: 4px;
+  margin-bottom: 5px;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    width: 100%;
+  }
+
+  .top-nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 16px;
+  }
+
+  .top-nav-ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .top-nav-a a {
+    padding: 10px;
+    text-decoration: none;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-size: 16px;
+    font-weight: 600;
+    color: rgb(9, 30, 66);
+  }
+
+  .greating {
+    display: none;
+  }
+
+  .menu {
+    display: none;
+    width: 0;
+    height: 0;
+  }
+
+  header {
+    width: 100%;
+    background-image: url(./assest/Header-llustration-desktop@2x.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    justify-items: center;
+  }
+
+  .main .mobile-h1 {
+    display: none;
+  }
+
+  .main .desk-h1 {
+    display: flex;
+    flex-direction: column;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: rgb(23, 43, 77);
+    margin-top: 50px;
+    align-items: center;
+  }
+
+  .support-text {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: rgb(23, 43, 77);
+    width: 60%;
+    margin-top: 24px;
+  }
+
+  .socialicon {
+    margin-top: 30px;
+    margin-right: 30px;
+  }
+
+  .arrow {
+    margin-bottom: 200px;
+    margin-right: 30px;
+  }
+
+  .recentwork .indicator {
+    display: none;
+   }
+
+  .text h2 {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: rgb(23, 43, 77);
+    }
+
+  section .recentwork {
+   display: flex;
+   flex-direction: column;
+   align-items: center;
+   width: 100%;
+   height: 1200px;
+  }
```

Comment:
- [ ] Required- Kindly use grid to layout cards. Grid is a better choice to create grills on the page. As a project requirement, you need to create 3 columns. Also, please change this title's width here, so it can be seen as the same as the picture below.
your design on larger screens: 
![image](https://user-images.githubusercontent.com/91486791/186753156-4669d61c-3d37-4dc8-a892-fdc95674092f.png)
it should be like this:
![image](https://user-images.githubusercontent.com/91486791/186753304-aa52b04a-f495-4fc7-abb9-80b0d3ebba69.png)